### PR TITLE
Power 4.6 Direct install, restricted install and RN

### PR DESCRIPTION
**Pull request by vikram-redhat**
@codyhoag now that https://github.com/openshift/openshift-docs/pull/26184/files has merged into the [enterprise-4](https://issues.redhat.com/browse/enterprise-4).6 branch, I am adding that into master. There were some conflicts, which I think I have resolved correctly, but please double check and merge if ok. You should be able to push this branch if there are any issues. Sorry I know it's extra work for you, but this one was very tricky, and I want to make sure I got this right so future updates can be done for Power and Z docs directly in master. 

@bobfuru I was unsure of the fixes I had to do due to this line that you recently updated: https://github.com/vikram-redhat/openshift-docs/commit/195ef0d7e18b8f283448dbb731ccee2efd208397#diff-812300ac04e88a61895332850e1e5f42004729ad650880e5cdf0b4b57f0490f1R56. These are here in this PR: https://github.com/openshift/openshift-docs/pull/26490/files#diff-812300ac04e88a61895332850e1e5f42004729ad650880e5cdf0b4b57f0490f1R56. Could you please double check it looks ok?

Thanks!

@SNiemann15 and @ktania46 - FYI. Future updates should be done in master.

### CODE REVIEW

**Review by codyhoag — COMMENTED** [review]
I found only one issue. I will make the change and force push.

**Comment by codyhoag** [comment]
@ktania46 can you review aae3b65a146016e3e4806978eec68c645db2237a (the second commit I made to make this `master` PR identical with the [4.6 PR](https://github.com/openshift/openshift-docs/pull/26184)) and make sure that's correct? I believe that content was being used by the bare metal installer docs, but wanted to check the rationale behind the edits.

If those edits are not needed, I can revert that commit. That change will also need to be made for 4.6 too, if that's the case, so the content is identical. Let me know :slightly_smiling_face: 

cc @SNiemann15 @vikram-redhat

**Comment by bobfuru** [comment]
> @bobfuru I was unsure of the fixes I had to do due to this line that you recently updated: [vikram-redhat@195ef0d#diff-812300ac04e88a61895332850e1e5f42004729ad650880e5cdf0b4b57f0490f1R56](https://github.com/vikram-redhat/openshift-docs/commit/195ef0d7e18b8f283448dbb731ccee2efd208397#diff-812300ac04e88a61895332850e1e5f42004729ad650880e5cdf0b4b57f0490f1R56). These are here in this PR: https://github.com/openshift/openshift-docs/pull/26490/files#diff-812300ac04e88a61895332850e1e5f42004729ad650880e5cdf0b4b57f0490f1R56. Could you please double check it looks ok?
@vikram-redhat I don't see where I actually updated that line, seems that it was already there? If we're both talking about line 56, this one:
`rd.neednet=1 console=ttysclp0 coreos.inst.install_dev=dasda coreos.inst.image_url=ftp://`

**Comment by ktania46** [comment]
lgtm/

**Comment by codyhoag** [comment]
@bobfuru can you confirm ade4b5f6ed2e991e4c87bf7da1f15d518625fdac? I saw you added in 281f70f32500c44d740bc2909932f454711a4fc0, so I suspect we should readd that for bare metal. Perhaps ifdefs are required for that?

**Comment by vikram-redhat** [comment]
Thanks @codyhoag and @bobfuru for making the fixes and reviewing this. I will wait for @bobfuru to confirm before merging this.

**Comment by bobfuru** [comment]
Thanks, @codyhoag, for helping me understand the impact here. OK to merge, @vikram-redhat. I'll then need to re-add what got removed in `modules/installation-user-infra-machines-static-network.adoc` and add ifdefs for IBM Z/Power version.

**Comment by codyhoag** [comment]
@vikram-redhat I squashed my commit that makes this PR identical with the `enterprise-4.6` branch. This should now be officially ready to merge. I'll let you do the honors :slightly_smiling_face:.

Bob will follow-up with the changes he summarized in https://github.com/openshift/openshift-docs/pull/26490#issuecomment-712283190 to get the content in order. We figured it was better to merge this PR so `master` and `enterprise-4.6` mirror each other, and then do the follow-up fix for both at the same time.

**Comment by vikram-redhat** [comment]
Okie dokie - merging! 

@openshift/team-documentation FYI - only because of possible impacts. If something in the install docs doesn't make sense, blame this PR (and me). ;) 

But doing this allows future updates to multi-arch docs easier and consistent.

**Comment by vikram-redhat** [comment]
Not to be CPed anywhere else.

File: modules/installation-user-infra-machines-static-network.adoc [hunk 1]
```diff
@@ -40,31 +40,31 @@ ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
 nameserver=4.4.4.41
 ----
 
-a|Specify multiple network interfaces by specifying multiple `ip=` entries.
-
+|Specify multiple network interfaces by specifying multiple `ip=` entries.
```
[the remark this answers is not in the document]
Comment (bobfuru) [reply modules/installation-user-infra-machines-static-network.adoc:33]: Thanks, @codyhoag. This change confuses me because it rewrites what was there before and I don't know what ifdefs to apply when I add the missing content back.

File: modules/installation-user-infra-machines-static-network.adoc [hunk 2]
```diff
@@ -40,31 +30,31 @@ ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
...
+|You can provide multiple DNS servers by adding a `nameserver=` entry for each server.
 a|
 ----
 nameserver=1.1.1.1
 nameserver=8.8.8.8
 ----
 
-a|Optional: Bonding multiple network interfaces to a single interface is supported using the `bond=` option. In these two examples:
+a|Optional: Bonding multiple network interfaces to a single interface is optionally supported
+using the `bond=` option.  In these two examples:
```
Comment (bobfuru) [inline modules/installation-user-infra-machines-static-network.adoc:57]: Why was this changed? It already prepends the sentence with `Optional:`.

**Comment by bobfuru** [comment]
Proposed fix to `modules/installation-user-infra-machines-static-network.adoc` file is introduced in https://github.com/openshift/openshift-docs/pull/26559.

File: modules/installation-user-infra-machines-static-network.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (bobfuru) [reply modules/installation-user-infra-machines-static-network.adoc:33]: Upon further review, I don't actually think ifdefs are necessary. Just a restore to the original content in this file.
